modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/pages/additional_issues_table.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module PdfConstruction
+    module HigherLevelReview
+      module Pages
+        class AdditionalIssuesTable
+          MAX_ISSUES_ON_FIRST_PAGE = 13
+
+          def initialize(pdf, form_data)
+            @pdf = pdf # Prawn::Document
+            @form_data = form_data
+          end
+
+          def build!
+            return pdf unless extra_issues?
+
+            pdf.start_new_page
+            # TODO: Make this into a table similar to NOD's extra issues table
+            pdf.text(extra_issues_text, inline_format: true)
+            pdf
+          end
+
+          private
+
+          attr_accessor :pdf, :form_data
+
+          def extra_issues?
+            form_data.contestable_issues.count > MAX_ISSUES_ON_FIRST_PAGE
+          end
+
+          def extra_issues_text
+            form_data.contestable_issues.drop(MAX_ISSUES_ON_FIRST_PAGE).map do |issue|
+              "Issue: #{issue['attributes']['issue']} - Decision Date: #{issue['attributes']['decisionDate']}"
+            end.join("\n")
```

Comment:
This part is different as well, with the lack of reverse, but i'm not sure it necessitates an entirely new class.

Reply:
Perhaps we can decide to reverse at initialization

Reply:
Same as my previous comment here - the eventual desire is for this file to become a tabular output instead of a list, I just didn't want to tackle that output in this PR.